rockspec

```diff
@@ -5,7 +5,8 @@ dependencies = {
   'luacheck >= 0',
   'busted  >= 0',
   'lua-cjson >= 0',
-  'ldoc >= 0'
+  'ldoc >= 0', 
+  'lua-resty-jwt >= 0'
```

Comment:
I think this is not necessary as development dependency because it is already a runtime dependency in the other rock spec.